### PR TITLE
Reply with the good template, logo and link !

### PR DESCRIPTION
**Pull request by djbuch**

### CODE REVIEW

**Comment by tchauviere** [comment]
:+1:

File: controllers/admin/AdminCustomerThreadsController.php [hunk 1]
```diff
@@ -399,7 +399,7 @@ public function postProcess()
                     $params = array(
                         '{reply}' => Tools::nl2br(Tools::getValue('reply_message')),
                         '{link}' => Tools::url(
-                            $this->context->link->getPageLink('contact', true),
+                            $this->context->link->getPageLink('contact', true, null, null, false,$ct->id_shop),
```
Comment (kijamve) [inline controllers/admin/AdminCustomerThreadsController.php]: Miss a space in the last comma.

**Comment by jeromenadaud** [comment]
Hi, thank you for your contribution, can you please make this PR on 1.6.1.x branch.
And me you have not noticed this: http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message

Best regards.

**Comment by jeromenadaud** [comment]
Merger in #3525

**Pull request closed without merging**